support-frontend/assets/components/checkoutBenefits/benefitsCheckList.tsx

```diff
@@ -145,9 +146,10 @@ export function BenefitsCheckList({
 							<span css={checkListTextItemCss}>
 								{item.isNew && (
 									<>
-										<NewBenefitPill />{' '}
+										<BenefitPill copy="New" />{' '}
 									</>
 								)}
+								{item.pill && <BenefitPill copy={item.pill} />}
```

Comment:
Maybe the `new` could be compressed into this line. Or even possibly `isNew` removed using your better approach of pill copy of `new` higher up?

Reply:
This component is used in a few places so I'll leave `isNew` as a prop, but I've refactored to simplify this bit